app/webpacker/src/stylesheets/tariff-custom.scss

```diff
@@ -78,3 +78,13 @@
 body {
   font-family: $govuk-font-family-gds-transport !important;
 }
+
+.govuk-breadcrumbs + .govuk-main-wrapper {
+  padding-top: 30px !important;
```

Comment:
this padding-top is 30px instead of 40px as the preceeding .govuk-breadcrumbs above has a margin-bottom of 10px 
![image](https://github.com/trade-tariff/trade-tariff-frontend/assets/127106895/4dc38dc2-04da-4a2c-adef-7c596f33d2e2)